data/json/uncraft/generic.json

```diff
@@ -52,6 +52,222 @@
     "qualities": [ { "id": "SAW_M", "level": 2 } ],
     "components": [ [ [ "scrap", 5 ] ] ]
   },
+  {
+    "result": "exodii_ingot_tin",
+    "type": "uncraft",
+    "activity_level": "MODERATE_EXERCISE",
+    "time": "16 h",
+    "tools": [ [ [ "angle_grinder", 500 ] ] ],
```

Comment:
Why use a specific tool, if there are multiple tools with grinding quality?

Reply:
The largest ingots are nearly a meter across. I'm all for adding other tools but doing it with hand tools isn't practical.

Reply:
Isn't angle grinder a hand tool, too?

What if stronger tools are added in the future? For example, in light industry you can find arc furnaces, and mines have unspecified heavy machinery, which may eventually include industrial grinders.

Reply:
"Hand tools" in the sense of an unpowered tool, using only the user's strength and any mechanical advantage. We may need to revisit qualities like metal sawing but I'd like to hold off on that until we work out a way to improve craft times based on that tool quality.